qlib/DebugCmdLine.qm

```diff
@@ -1238,15 +1238,21 @@ public namespace DebugCmdLine {
                             showFullException ? printf("%y\n", ex) : printf("%s: %s\n", ex.err, ex.desc);
                         }
                     } else {
-                        if (last_param < args.size())
+                        if (elements keys cmd.variants > 1) {
```

Comment:
please just use `element cmd.variants` which operates directly on the hash (returns the number of hash keys) - `elements keys variants` creates a temporary list and then returns the elements in the list - or you can use `cmd.variants.size()`